backend/lib/components/Auth0.js

```diff
@@ -105,10 +106,120 @@ const changePassword = async (token, email) => {
   }
 };
 
+const _getUserDetails = async (token, userId) => {
+  try {
+    const res = await axios.request(
+      `${AUTH_DOMAIN}/api/v2/users/${userId}`,
+      getAuthHeaders(token),
+    );
+    return res.data;
+  } catch (err) {
+    return wrapError(err);
+  }
+};
+
+const _updateUser = async (token, userId, payload) => {
```

Comment:
```suggestion
const _updateUser = async (token, userId, payload) => {
```

Not sure what you're planning to use this for but we actually are just about to implement something very similar here:

https://github.com/FightPandemics/FightPandemics/blob/86883fe123b01991c6488fa966e9c68a331267db/backend/lib/components/Auth0.js#L88

It is very close to getting ready to merge so maybe we should wait on that first?

Reply:
this is for merging the meta-data only, since we're removing that part, I will also remove this one